src/LayerEditor/data/config.py

```diff
@@ -21,6 +21,11 @@ class _Config:
     """Count of recent files"""
 
     CONFIG_DIR = os.path.join(base_cfg.__config_dir__, 'LayerEditor')
+    """Dir where config file is saved"""
+
+    geomop_root = os.path.dirname(os.path.dirname(
+        os.path.dirname(os.path.realpath(__file__))))
+    """Path to the root directory of the GeoMop installation."""
```

Comment:
I want an absolute path of src directory. `__file__` returns absolute path so join will not work. I changed it to:
`geomop_root = os.path.dirname(os.getcwd())`
So that should be more readable.

Reply:
But os.getcwd dependes on the working directory of the system.

Reply:
Thank you for noticing that would create the same issue that was in Genie with colormaps. Changed to: Path(__file__).parent.parent.parent. Hopefully its still readable.